dashboard/app/views/blocks/edit.html.haml

```diff
@@ -12,8 +12,8 @@
       url: @block.new_record? ? blocks_path : block_path(id: @block.name),
       html: {id: 'edit-block-form', method: @block.new_record? ? 'post' : 'patch'},
     ) do |f|
-      = f.label 'Level Type'
-      = f.select :level_type, ['GamelabJr', 'Maze']
+      = f.label 'Pool Name'
+      = f.select :pool, Block.all_pool_names + [params[:pool]].compact
```

Comment:
Is there a GUI for adding a new pool, or do you just have to add this query param?

Reply:
Only the query param, for now.  I think we'll want a separate model+controller to manage these pools (and what libraries / shared functions / app type they're associated with).